parquet/src/file/metadata.rs

```diff
@@ -303,9 +309,18 @@ impl RowGroupMetaData {
             let cc = ColumnChunkMetaData::from_thrift(d.clone(), c)?;
             columns.push(cc);
         }
+        let sorting_columns: Option<Vec<SortingColumnMetaData>> = match rg.sorting_columns
```

Comment:
```suggestion
        let sorting_columns = match rg.sorting_columns
```

You shouldn't need to type hint this